src/main/java/com/mentoring/ecommerce/adapter/in/web/ProductController.java

```diff
@@ -17,10 +18,18 @@
 public class ProductController {
 
     private SaveProductUseCase saveUseCase;
+    private FindProductUserCase findUseCase;
     private ProductMapper productMapper;
 
     @PostMapping()
     public void saveProduct(ProductReq request) {
         saveUseCase.saveProduct(productMapper.toDomain(request));
     }
+
+    @GetMapping()
+    public ResponseEntity<List<ProductRes>> findAllProducts() {
```

Comment:
`final List<ProductRes>`
ProductRes -> ProductResponse

Reply:
I think is not possible to make a _type_ as final: `ResponseEntity<final List<ProductRes>>`.
Shouldn't `final` be used only in attributes, methods and variables?

Reply:
Oh, you're right! My bad!